share/spice/news/news.js

```diff
@@ -55,6 +55,7 @@
             story.title = story.title.replace(/<b>|<\/b>|:/g, "");
             if (DDG.isRelevant(story.title, skip)) {
                 setSourceOnStory(story);
+                story.sortableDate = parseInt(story.date);
```

Comment:
trying to do this only when necessary, and only once for each element instead of when comparing